modules/swagger-codegen/src/main/resources/go-server/main.mustache

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	sw "./{{apiPath}}"
```

Comment:
ah, thanks for explaining, since everyone will have a different fully-qualified path, I will take the warning approach.(code has been updated.)